sklearnex/__init__.py

```diff
@@ -50,12 +51,13 @@
     "unpatch_sklearn",
     "utils",
 ]
+onedal_iface_flag = os.getenv("OFF_ONEDAL_IFACE", "0")
+if onedal_iface_flag == "0":
```

Comment:
```suggestion
onedal_iface_flag = os.getenv("OFF_ONEDAL_IFACE", None)
if onedal_iface_flag:
```

Reply:
Then In this case, if `OFF_ONEDAL_IFACE=0` is set then still it will try to import `onedal` which again can cause error.

Reply:
Then I would also recommend changing here: https://github.com/intel/scikit-learn-intelex/blob/main/setup.py#L75